### PR TITLE
fix PyDocStyle checks

### PR DESCRIPTION
**Pull request by dannyl1u**
related: https://github.com/apache/airflow/issues/40567
PT004 | Fixture does not return anything, add leading underscore
Reduces PT004 errors from 345 to 318.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/www/views/test_views_cluster_activity.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def make_dag_runs(dag_maker, session, time_machine):
     session.flush()
 
 
-@pytest.mark.usefixtures("freeze_time_for_dagruns", "make_dag_runs")
```
Comment (ferruzzi) [inline tests/www/views/test_views_cluster_activity.py:102]: [Non-blocking] Not expecting you to change this, but I wonder why they used `usefixtures` here and if we can move those fixtures into the test method signatures like we do (almost?) everywhere else.

Feel free to resolve this and merge after a second approval.

**Review by ferruzzi — APPROVED** [review]
LGTM

File: tests/www/views/test_views_cluster_activity.py (hunk 1 above)
Comment (dannyl1u) [reply tests/www/views/test_views_cluster_activity.py:102]: I'll look into this, if we can change it I'll do it in a separate PR

**Review by jscheffl — APPROVED** [review]
views.py will be removed in 3.0 do actually this PR improves testing in almost dead code... but anyway still improves :-D

**Pull request merged**